raijin_scripts/execute_ingest/run

```diff
@@ -26,22 +24,35 @@ while [[ $# -gt 0 ]]; do
     shift
 done
 
-
-echo Executing ingest for "${YEAR}" "${PRODUCT}"
-echo Loading module "${DEA_MODULE}"
-
-module use /g/data/v10/public/modules/modulefiles
-
-module load "${DEA_MODULE}"
-
-cd /g/data/v10/work/ingest/
-
-SUBMISSION_LOG=/g/data/v10/work/ingest/ingest-submission-${PRODUCT}-$(date '+%F-%T').log
+WORKDIR=/g/data/v10/work/ingest/"${PRODUCT}"/$(date '+%FT%H%M')
+SUBMISSION_LOG="${WORKDIR}"/ingest-submission-${PRODUCT}-$(date '+%F-%T').log
 JOB_NAME="${YEAR}_${PRODUCT}"
 
-echo "Logging job: ${JOB_NAME} into: ${SUBMISSION_LOG}"
+mkdir -p "${WORKDIR}"
+echo "Start time: " "$(date '+%F-%T')" > "$SUBMISSION_LOG"
 
 # Launch the submission in the background, outputting results to a log file
+cd "${WORKDIR}"
 nohup "$SHELL" > "$SUBMISSION_LOG" 2>&1 <<EOF &
+
+echo "Logging job: ${JOB_NAME} into: ${SUBMISSION_LOG}"
+
+echo ""
+echo Loading module "${DEA_MODULE}"
+module use /g/data/v10/public/modules/modulefiles
+module load "${DEA_MODULE}"
+echo ""
+
+# Check if we can connect to the database
+datacube -vv system check
+
+# Read agdc datasets from the database before ingest process
+echo ""
+echo "**********************************************************************"
+echo "Read previous agdc_dataset product names and count before ingest process"
+psql -h agdcdev-db.nci.org.au -p 6432 -d datacube -c 'select name, count(*) FROM agdc.dataset a, agdc.dataset_type b where a.dataset_type_ref = b.id group by b.name'
```

Comment:
This is still counting in the dev database, and still isn't logging which database it's running on...